src/test/java/io/opensaber/views/TransformerTest.java

```diff
@@ -0,0 +1,76 @@
+package io.opensaber.views;
+
+import static org.junit.Assert.assertEquals;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import java.io.ByteArrayOutputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import org.junit.Test;
+
+public class TransformerTest {
+
+    private Transformer transformer = new Transformer();
+  
+    @Test
+    public void testTransformForPersonVT() throws JsonProcessingException, IOException{
+        String personJson = "{\"Person\": " +
+                "               {\"nationalIdentifier\":\"nid823\"," +
+                "                \"firstName\":\"Ram\"," +
+                "                \"lastName\":\"Moorthy\"," +
+                "                \"gender\":\"MALE\"," +
+                "                \"dob\":\"1990-12-10\"}}";
+        ObjectNode personNode = (ObjectNode) new ObjectMapper().readTree(personJson);
+        String viewTemplateJson = readFileContent("personVT1.json");
+        ViewTemplate viewTemplate = new ObjectMapper().readValue(viewTemplateJson, ViewTemplate.class);
+
+        JsonNode actualnode = transformer.transform(viewTemplate, personNode);  
+        JsonNode expectedNode = new ObjectMapper().readTree("{\"Person\":{\"NAME\":\"Ram\",\"lastName\":\"Moorthy\",\"Name in passport\":\"Moorthy, Ram\"}}");
+        assertEquals(expectedNode, actualnode);
+
+    }
+    
+    @Test
+    public void testTransformForMathVT() throws JsonProcessingException, IOException{
+        String mathProblem = "{\"Math\": " +
+                "               {\"a\": 5," +
+                "                \"b\": 2 }}";
+        ObjectNode node = (ObjectNode) new ObjectMapper().readTree(mathProblem);
```

Comment:
this is duplicate code repeated in tests...if you've to add a third test, you must think of creating functions for this.